.gitignore

```
@@ -21,3 +21,5 @@ tests/__pycache__/
 .pytest_cache/
 *.ipynb
 .DS_Store
+.vscode/settings.json
+.vscode/launch.json
```

```suggestion
.vscode/
```

this will ignore all files in the `.vscode` folder. Here is a short article about the [philosopy of `.gitignore`](https://dev.to/koddr/quick-how-to-stop-copy-paste-and-make-gitignore-globally-282i)

---

also note that this is a "suggestion" and can be committed directly through the github gui. (so no need to make the change again yourself)